resources/views/layouts/navigation.blade.php

```diff
@@ -12,14 +12,60 @@
 
                 <!-- Navigation Links -->
                 <div class="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
+
+                    <!-- Update this so route goes to the result/search screen -->
```

Comment:
I did not quite understand this. Why are we adding substituting the search page? 

This one is also redirecting to the dashboard, which does not make much sense. 

Probably better to keep the dashboard and add an additional tab for the search when we implement that.

Reply:
Will consider adding an addition tab later for search bar implementation 

For now, changed tag to 'Movies' and now routes to 'movies.index' for the time being